src/test/resources/application.properties

```diff
@@ -24,6 +24,7 @@ quarkus.kafka-streams.application-id=kafkastreams-example-app
 %donator-extractor.quarkus.kafka-streams.topics=kafkastreams.source-topic,kafkastreams.target-topic
 %donator-enrichment.quarkus.kafka-streams.topics=kafkastreams.donation,kafkastreams.collector,kafkastreams.donator
 quarkus.kafka-streams.schema-registry-url=<overriden>
+quarkus.kafka-streams.default.key.serde=org.apache.kafka.common.serialization.Serdes$StringSerde
```

Comment:
to avoid and fix problem: https://blog.adafycheng.dev/orgapachekafkacommonconfigconfigexception-please-specify-a-key-serde-or-set-one-through-streamsconfigdefaultkeyserdeclassconfig